xdsl/dialects/gpu.py

```diff
@@ -45,6 +45,7 @@
     result_def,
     var_operand_def,
 )
+from xdsl.irdl.irdl import opt_prop_def, prop_def
```

Comment:
```suggestion
from xdsl.irdl import opt_prop_def, prop_def
```

Isn't this the way to do this?

Reply:
There are more of these.

Reply:
Thanks, will do a pass!